Include/cpython/pystate.h

```diff
@@ -83,14 +83,20 @@ struct _ts {
         unsigned int bound_gilstate:1;
         /* Currently in use (maybe holds the GIL). */
         unsigned int active:1;
+#ifdef Py_GIL_DISABLED
+        /* Currently holds the GIL. */
+        unsigned int holds_gil:1;
+#else
+        unsigned int _unused:1;
+#endif
```

Comment:
I'd define and track this in both the default build and the free-threaded build because it's cheap to update and may provide a useful debugging aid.